pro-client/enable_realtime_kernel.rst

```diff
@@ -1,92 +1,146 @@
 .. _manage-realtime:
 
-How to manage real-time kernel
+How to enable Real-time Ubuntu
 ******************************
 
-Pre-requisites
-==============
+`Real-time Ubuntu <realtime_>`_ is supported on specific Ubuntu releases. To
+ find out more about the supported Ubuntu versions and what kernel variants
```

Comment:
In the "Read the Docs build" the line 7 is indented. Do we want to keep the text in the next line or right after "To" in the previous sentence?

Reply:
No, that's a great catch! It's not supposed to be indented. I'll fix that :)